app/views/orders/cart.html.erb

```diff
@@ -0,0 +1,86 @@
+<div class="row">
+  <h1 class="header-box">
+    Your Cart
+  </h1>
+</div>
+<div class="row">
+  <div class="col-sm-12">
+    <% if @current_order == nil %>
+      <h3 class="emptycart"> Your cart is empty </h3>
+      <h3 class="emptycart"> <%= link_to "Go shopping!", products_path %> </h3>
+    <% else %>
+  </div>
+</div>
+<div class="row">
+  <div class="col-12-sm">
+    <div class="table-master">
+    <table class='table table-hover'>
```

Comment:
Be sure to be consistent with your quotes. I know it seems really nit picky, but when writing production code it's important.